arxiv-matplotlib.ipynb

```diff
@@ -175,7 +175,6 @@
     "cluster = coiled.Cluster(\n",
     "    n_workers=100,\n",
     "    backend_options={\"region\": \"us-east-1\"},  # faster and cheaper\n",
```

Comment:
We could use the new `region=` kwarg here too (not a blocking comment, just a note)